File: C/Chapter 3 - Control Statement/2K221008/patteren_star_ascending.c
//Loops:

//Pattern Programs

/*

Ex. Star Pattern - Ascending

*
* *
* * *
* * * *
* * * * *

*/

#include<stdio.h>
void main()
{
    int count,i,j;

    printf("Enter the count: ");
    scanf("%d",&count);
    printf("\n");

/*
    while(i <=count){
            j=1;
        while (j <= i){
            printf("* ");
            j++;
        }
        printf("\n");
        i++;
    }
*/
    for(i = 1; i <= count; i++){
        for (j=1;j<=i;j++){
            printf("* ");
        }
        printf("\n");
    }
}

File: Back End/Python/Chapter 05 - Control Statements/2K221229/patteren_star_ascending.py
# Control Statements
# Loops:
#
# Pattern Programs
#
# Ex. Star Pattern - Ascending
#
# *
# * *
# * * *
# * * * *
# * * * * *

num = int(input("Enter the limit: "))

for i in range(1, num + 1, 1):
    for j in range(1, i + 1, 1):
        print("* ", end=' ')
    # print(end='\n\\n')
    print(end='\n')
